### PR TITLE
[WFCORE-6834] wildfly-elytron-integration jar duplicated in server modules

### PR DESCRIPTION
**Pull request by yersan**
Jira issue: https://issues.redhat.com/browse/WFCORE-6834

### CODE REVIEW

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks @yersan. The change is fine but since this is really strange stuff I think a comment is needed.

File: core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/elytron/jaas-realm/main/module.xml [hunk 1]
```diff
@@ -48,5 +40,10 @@
         <module name="org.jboss.logmanager"/>
         <module name="org.wildfly.common"/>
         <module name="org.wildfly.security.elytron-private" export="true"/>
+        <module name="org.wildfly.extension.elytron" export="true">
+            <exports>
+                <exclude path="org/wildfly/extension/elytron/*"/>
```
[the remark this answers is not in the document]
Comment (bstansberry) [reply core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/elytron/jaas-realm/main/module.xml:45]: When you add a comment please don't block on me to re-review. I will if I notice, but we're close to the release so I don't want to delay your work on it.

[the remark this answers is not in the document]
Comment (yersan) [reply core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/elytron/jaas-realm/main/module.xml:52]: Done

**Comment by yersan** [comment]
Thanks @ivassile @bstansberry

**Pull request merged**